lib/modules/manager/helmfile/extract.ts

```diff
@@ -39,8 +41,9 @@ export async function extractPackageFile(
   // Record kustomization usage for all deps, since updating artifacts is run on the helmfile.yaml as a whole.
   let needKustomize = false;
   try {
-    // TODO: use schema (#9610)
     docs = parseYaml(content, null, {
+      customSchema: documentSchema,
+      failureBehaviour: 'filter',
```

Comment:
I don't remember, does it log a message for invalid documents?

Reply:
Yes it logs in such scenarios https://github.com/renovatebot/renovate/blob/main/lib/util/yaml.ts#L52